.github/workflows/docker-ort-runtime-ext.yml

```diff
@@ -15,23 +15,25 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
-name: Mega Docker extended image
+name: Docker extended image
```

Comment:
This does more than just changing the schedule, which does not look right.

Reply:
Solved